src/Components/Contributors/ContributorModal.tsx

```diff
@@ -80,6 +97,27 @@ const StyledContributorFooter = styled.div`
 
 const NorwegianCountryCode = 'NO';
 
+const doesContributorHaveNorwegianAffiliation = (contributor: ContributorWrapper) =>
+  contributor.imported?.affiliations?.some((inst) => inst.countryCode === NorwegianCountryCode);
+
+const NumberOfContributorsToShow = 30;
+
+export const NumberOfContributorsToDefineMonsterPost = 100;
+
+const contributorHasErrors = (contributor: ContributorWrapper): boolean =>
+  contributor.toBeCreated?.affiliations?.length === 0 ||
+  contributor.toBeCreated?.first_name === '' ||
+  contributor.toBeCreated?.surname === '' ||
+  contributor.toBeCreated?.first_name?.length > MaxLengthFirstName ||
+  contributor.toBeCreated?.surname?.length > MaxLengthLastName;
```

Comment:
Fant ut det var bedre å ha denne logikken flere steder enn å lage noe alt for komplekst system som holder det globalt